### PR TITLE
Code to fix lvsetup.py for multiple distros

### PR DESCRIPTION
**Pull request by hathyaga**
Fix to handle xfs and btrfs filesystems

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
[lvsetup_joblog.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/803307/lvsetup_joblog.txt)

File: io/disk/lvsetup.py [hunk 1]
```diff
@@ -45,11 +46,19 @@ def setUp(self):
...
         """
+        pkg = " "
+        smm = SoftwareManager()
         self.disk = self.params.get('disk', default=None)
         vg_name = self.params.get('vg_name', default='avocado_vg')
         lv_name = self.params.get('lv_name', default='avocado_lv')
         self.lv_size = self.params.get('lv_size', default='1G')
-        self.fs_name = self.params.get('fs', default='ext4')
+        self.fs_name = self.params.get('fs', default='ext4').lower()
+        if self.fs_name.lower() == 'xfs':
```
Comment (PraveenPenguin) [inline io/disk/lvsetup.py]: self.fs_name should be sufficient as you already did  lower conversion 

`self.fs_name = self.params.get('fs', default='ext4').lower()`

File: io/disk/lvsetup.py [hunk 2]
```diff
@@ -45,11 +46,19 @@ def setUp(self):
...
+            pkg = 'xfsprogs'
+        if self.fs_name.lower() == 'btrfs':
```
Comment (PraveenPenguin) [inline io/disk/lvsetup.py]: same here as well

**Comment by hathyaga** [comment]
Closing this. New PR: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/289

**Pull request closed without merging**